pandas/core/ops/__init__.py

```diff
@@ -51,6 +51,21 @@
 import pandas.core.common as com
 import pandas.core.missing as missing
 
```

Comment:
would it be worth it to move this to ``ops.py`` and then import only public functions to ``__init__.py``?

Reply:
I think before long, yes.  ATM I've got several branches touching ops so I decided to keep this simple for rebasing purposes.

Reply:
When moving things, I much prefer moving things at once to a file you think to have it longer term. Moving files are annoying in git history ;)